### ISSUE TITLE
508 Defect 3, form validation, able to enter dates which a reasonable person

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#22909]
Label: 22685

    - [508-defect-3]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3

Issue Title: 508 Defect 3, form validation, able to enter dates which a reasonable person would conclude are in error.

 ## Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
In order to prevent forms from being submitted and disqualified because of clerical errors I.E. 1920 rather than 2020, it is helpful to have form validation rules which mitigating the entering of dates which a reasonable person would conclude aren't possible.
<h2>Details</h2>
<ul>
<li>When entering my work history, I am able to enter an end date which precedes my start date and proceed to the next step. </li>
<li>When entering my work history, I am able to enter a date in the year 1900</li>
<li>When asked for my and my spouses jobs within the last two years, I am able to enter jobs whose end date is well prior to two years ago.</li>
<li>I am able to enter a bankrupcy date in the year 1900, </li>
</ul>
<h2>Acceptance Criteria</h2>
- [ ] When a date is entered which a reasonable person would consider impossible or which is out of the range stated in the question, the user is presented with a clean error message spoken by JAWS and the form cannot be submitted.

**Comment by denisecoveyduc** [comment]
@joshkimux @AngelaFowler82  I am doing a little GH clean up and wondered if you could just take a quick look at this issue for me to validate that this is still a concern? Since (and before I joined the team) the FSR has gone through many changes and this may or may not have already been addressed. Thanks in advance for checking it out for us.

**Comment by denisecoveyduc** [comment]
Shifting this over to FSR Tech Debt Epic to ensure we address it going forward.

**Comment by mgleo** [comment]
@HeatherRienks @Tom-Davis I believe this can be closed as dates are manually entered in a text input field, not selected

**Comment by Tom-Davis** [comment]
The product has evolved beyond the need for this - closing as no longer needed.
